.github/workflows/create_minor_release_PR_commit.yaml

```diff
@@ -0,0 +1,32 @@
+name: Create minor release PR commit
```

Comment:
This is the release PR to main, right? Let's name it accordingly, something like "Create PR to main branch for minor release".

Reply:
Edited names of steps accordingly.